src/site/content/en/blog/trust-tokens/index.md

```diff
@@ -238,59 +255,54 @@ fetch('issuer.example/.well-known/trust-token', {
-information about which sites its users visit, but can't link issuance with
-redemption: when a user redeems a token, the issuer can't tell the token apart
-from other tokens it has created. However, trust tokens currently do not exist
-in a vacuum: there are other ways an issuer could currently—in theory—join a
-user's identity across sites, such as third-party cookies and covert tracking
-techniques. It is important for sites to understand this ecosystem transition as
-they plan their support. This is a general aspect of the transition for many
-Privacy Sandbox APIs, so not discussed further here.
+Tokens are designed to be 'unlinkable'. An issuer can learn aggregate information about which sites 
+its users visit, but can't link issuance with redemption: when a user redeems a token, the issuer 
+can't tell the token apart from other tokens it has created. However, trust tokens currently do not 
+exist in a vacuum: there are other ways an issuer could currently—in theory—join a user's identity 
+across sites, such as third-party cookies and covert tracking techniques. It is important for sites 
+to understand this ecosystem transition as they plan their support. This is a general aspect of the 
+transition for many Privacy Sandbox APIs, so not discussed further here.
 
 ### Security considerations
 
-**Trust token exhaustion:** a malicious site could deliberately deplete a user's
-supply of tokens from a particular issuer. There are several mitigations against
-this kind of attack, such as enabling issuers to provide many tokens at once, so
-users have an adequate supply of ensuring browsers only ever redeem one token
-per top-level page view.
+**Trust token exhaustion:** a malicious site could deliberately deplete a user's supply of tokens 
+from a particular issuer. There are several mitigations against this kind of attack, such as 
+enabling issuers to provide many tokens at once, so users have an adequate supply of ensuring 
+browsers only ever redeem one token per top-level page view.
 
-**Double-spend prevention:** malware might attempt to access all of a user's
-trust tokens. However, tokens will run out over time, since every redemption is
-sent to the same token issuer, which can verify that each token is used only
-once. To mitigate risk, issuers could also sign fewer tokens.
+**Double-spend prevention:** malware might attempt to access all of a user's trust tokens. However, 
+tokens will run out over time, since every redemption is sent to the same token issuer, which can 
+verify that each token is used only once. To mitigate risk, issuers could also sign fewer tokens.
 
 ### Request mechanisms
 
-It might be possible to allow for sending RRs outside of `fetch()`, for example
-with navigation requests. Sites might also be able to include issuer data in
-HTTP response headers to enable token redemption in parallel with page
-loading.
+It might be possible to allow for sending RRs outside of `fetch()`, for example with navigation 
```

Comment:
Does RR stand for redemption request?

Reply:
Oops — good spot. Changed to 'redemption record'.